utils/parametric/_library_client.py

```diff
@@ -42,17 +42,18 @@ def __init__(self, url: str, timeout: int, container: Container):
         self.container = container
 
         # wait for server to start
-        self._wait(timeout)
+        self.wait(timeout, True)
 
-    def _wait(self, timeout):
+    def wait(self, timeout, reload):
         delay = 0.01
         for _ in range(int(timeout / delay)):
             try:
                 resp = self._session.get(self._url("/non-existent-endpoint-to-ping-until-the-server-starts"))
                 if resp.status_code == 404:
                     break
             except Exception:
-                self.container.reload()
+                if reload:
```

Comment:
For which reason you may not want to reload?

Reply:
In APMLibrary.is_alive() we want to check whether the parametric app is running. This is useful when validating the behavior of the trace/crash endpoint. I can also see a world where we want to use this method in more crash tracking tests or as a quick health check.

Reply:
But if the container is stopped, there is no chance that the app is running, so you'll wait for nothing no? Or do I miss something ?

Reply:
I think `_wait` may not be a good fit for `is_alive`, as it's prupose is to be a healtcheck, so you'll had to handle some extra stuff. May a suggest a dedicated function that : 

1. check the container status
2. check that the endpoint respond 404
3. returns True if 1. and 2. are successful

Reply:
Done: 095a58ab2aa6eac7b4017f1d66a51a162e04622d